pinax/stripe/actions/accounts.py

```diff
@@ -0,0 +1,223 @@
+import stripe
+
+from .. import models
+from .. import utils
+import datetime
+from . externalaccounts import sync_bank_account_from_stripe_data
+
+
+def create(user, country, **kwargs):
+    """
+    Create an Account.
+
+    Args:
+        country: two letter country code for where the individual lives
+
+    Returns:
+        a pinax.stripe.models.Account object
+    """
+    kwargs['country'] = country
```

Comment:
I'd like to stay consistent with rest of code base and not use single-quotes, but instead use double-quotes.